modules/samples/oauth2/playground2/src/main/resources/playground2.properties

```diff
@@ -0,0 +1,17 @@
+#We can get these from Identity Server OAuth2 application configs.
+ConsumerKey=
+ConsumerSecret=
+#OIDC scope value as a default. Can configure more than one in ',' separator.
+Scope=openid
+#Callback URL endpoint that is configured in Identity Server for this OAuth2 application.
+CallbackURL=http://localhost:8080/playground2/oauth2client
+
+#Identity Server endpoint details.
+IdentityServerHostName=localhost
+IdentityServerPort=9443
+AuthorizeEndpointContext=/oauth2/authorize
+AccessTokenEndpointContext=/oauth2/token
+LogoutEndpointContext=/oidc/logout
+UserInfoEndpointContext=/oauth2/userinfo
+#client_id value read from ConsumerKey value and build the context ex: '/oidc/checksession?client_id=hf87987fyds8fs8f87ds8fs'
+SessionIFrameEndpointContext=/oidc/checksession?client_id=
```

Comment:
Can we remove the 'client_id' parameter as well and build it from the code?

Reply:
Done